src/components/Budget/BudgetForm/BudgetInformationFields.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Col,
+  Row,
+  KeyValue,
+  TextField,
+  Datepicker,
+} from '@folio/stripes/components';
+import {
+  FieldSelect,
+} from '@folio/stripes-acq-components';
+import { BUDGET_STATUSES_OPTIONS } from '../constants';
+
+const BudgetInformationFields = ({
+  fiscalEnd,
+  fiscalStart,
+}) => (
+  <Row>
+    <Col xs={3}>
+      <Field
+        component={TextField}
+        fullWidth
+        label={<FormattedMessage id="ui-finance.budget.name" />}
+        name="name"
+      />
+    </Col>
+
+    <Col xs={3}>
+      <Datepicker
+        label={<FormattedMessage id="ui-finance.budget.allocationDate" />}
+        disabled
+      />
+    </Col>
+
+    <Col xs={3}>
+      <FieldSelect
+        dataOptions={BUDGET_STATUSES_OPTIONS}
+        id="invoice-status"
+        label={<FormattedMessage id="ui-finance.budget.status" />}
+        name="budgetStatus"
+        required
+      />
+    </Col>
+
+    <Col xs={3}>
+      <Datepicker
```

Comment:
maybe it makes sense to use `FieldDatepicker` from stripes-acq-components?

Reply:
I think no. Because this fields are disabled and we do not get them from the response. also we can't send this field with request to back-end.